apps/core/app/(landing)/_components/addtoCard.tsx

```
@@ -0,0 +1,110 @@
+        {cards.map((card) => (
+          <Card
+            title={card.title}
+            price={card.price}
+            key={card.id}
+            id={card.id}
+            onRemove={handleRemove}
+          />
+        ))}
+
+        {activeTemp.map((card) => (
+          <CardSkeleton />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)